spa/src/pages/ObjectProperties.vue

```diff
@@ -245,7 +245,7 @@ methods: {
       if(this.walletBalance >= this.price){
         try{
             await this.objectProperties();
-            if(!this.price){
+            if(!this.price && this.price !== 0){

```

Comment:
Can we change this to less than 0?  This way we catch any accidental negative prices as not for sale.

Reply:
I will look at it. Any negative value input into the price becomes positive in the api so it shouldn't ever happen, but wouldn't hurt to add it on the front end as an added precaution.